src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -99,18 +97,21 @@
     @MockBean
     AuthTokenGenerator authTokenGenerator;
 
-    protected static final String ACCESS_IS_DENIED_ERROR_MESSAGE = "Access is denied";
-
     @Autowired
     ObjectMapper objectMapper;
 
     @Autowired
     protected CaseWorkerIdamRoleAssociationRepository roleAssocRepository;
 
+    @Autowired
+    Flyway flyway;
```

Comment:
why @flyway needed

Reply:
The overall approach is to use single db instance and reuse it. We are using Flyway to clean up db schema and re-apply migrations. 
Please let me know if we need to discuss it n detail.